R/targets_utils.R

```diff
@@ -138,7 +138,14 @@ make_shared_grids <- function() {
     ),
     tidyr::expand_grid(
       forecaster = "flatline_fc",
-      ahead = 1:7
+      ahead = c(1:7, 14, 21, 28)
+    ),
+    tidyr::expand_grid(
+      forecaster = "smoothed_scaled",
+      trainer = c("quantreg"),
+      ahead = c(1:7, 14, 21, 28),
+      lags = list(list(c(0, 3, 5, 7, 14), c(0),), c(0, 7, 14)),
```

Comment:
suggest: try some larger lags than 14.

todo: I don't know what predictor selections these lag selections line up to... is there a way to explicitly specify?  Or document in comment.

suggest: try out the dev10v4s lag selection to help contextualize.

Reply:
At the moment the lags are applied to all variables simultaneously. We  may be able to use a named list? I'll have to experiment. I'm also not 100% sure that `list(c(0, 3, 5, 7, 14), c(0),)` will work as written; I *think* this should apply `c(0,3,5,7,14)` to `hhs_m7` and `chng_m7` and `c(0)` to `hhs_sd28` and `chng_sd28`, but I'm not sure.

Reply:
confirmed that this works as intended, and added some lags longer than 14 in a separate list